bitinformation_pipeline/bitinformation_pipeline.py

```diff
@@ -405,6 +405,186 @@ def plot_bitinformation(bitinfo):
     return fig
 
 
+def get_prefect_flow(paths=[]):
+    """
+    Create prefect.Flow for bitinformation_pipeline bitrounding paths.
+
+    1. Analyse bitwise real information content
+    2. Retrieve keepbits
+    3. Apply bitrounding with `xr_bitround`
+    4. Save as compressed netcdf with `to_compressed_netcdf`
+
+    Many parameters can be changed when running the flow `flow.run(parameters=dict(chunk="auto"))`:
+    - paths: list of Paths
+        paths to be bitrounded
+    - analyse_paths: str or int
+        which paths to be passed to `bp.get_bitinformation`. choose from ["first_last", "all", int], where int is interpreted as stride, i.e. paths[::stride]. Defaults to "first".
+    - enforce_dtype : str or None
+        Enforce dype for all variables. Currently `get_bitinformation` fails for different dtypes in variables. Do nothing if None. Defaults to None.
```

Comment:
`flow.run(parameters=dict(enforce_dtype="float32"))` circumvents error for files with vertices as data_vars as float64 when other vars are float32, as in ICON or MPIOM.